### DISCUSSION TITLE
Can ExplorerPatcher be used to suppress the taskbar when explorer.exe launches?

### DISCUSSION THREAD

**Discussion by jazir555** [valinet/ExplorerPatcher#2786] [General]
My goal here is to set Steam Big Picture as the default shell so nothing is displayed until the big picture UI launches, then have explorer.exe start up in the background so the "exit to desktop" menu item functions as intended. 

The main functionality I want already works.

A batch script sets Steam with the Big Picture launch option to the default shell so it launches at boot without explorer displaying(screen remains black until Big Picture launches). The script creates a second batch script which is launched via a scheduled task which sets explorer to the default shell so it can be launched in the background after Steam Big Picture launches. Another 10 seconds after explorer launches, Steam is reset to the default shell so that Big Picture opens at every boot up.

The issue is that the taskbar displays over the big picture UI at the bottom of the screen for ~1 second (the rest of Big Picture remains displayed) when explorer.exe is launched, and then the taskbar disappears, refocusing onto Steam Big Picture (Big Picture remains fullscreen).

I don't want the taskbar to appear at all until the user exits Big Picture. Can the taskbar be suppressed via ExplorerPatcher so that the taskbar UI element does not display over Big Picture when explorer launches?

**Comment by pyrates999** [comment]
No.  But you can look at windows kiosk mode.

https://www.techtarget.com/searchenterprisedesktop/definition/kiosk-mode
